api/app.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express")
+
+const { getCategories } = require("./controllers/games.controllers.js")
+
+const app = express();
+app.use(express.json())
+
+app.get("/api/categories", getCategories)
+
+app.use((err, req, res, next) => {
+    console.log(err, "internal server error")
```

Comment:
remove console.log before merge

Reply:
the console.log is intentional for error handling, not for debugging so i'd like to leave it in the project generally, it only logs when there is an error - is that okay?